blocks/loader.ts

```diff
@@ -190,31 +200,18 @@ const wrapLoader = ({
         const url = new URL("https://localhost");
         url.searchParams.set("resolver", loader);
 
-        const span = tracer.startSpan("object-hash", {
-          attributes: {
-            props_length: JSON.stringify(props).length,
-          },
-        });
-
-        try {
-          const hashedProps = hash(props, {
-            ignoreUnknown: true,
-            respectType: false,
-            respectFunctionProperties: false,
-            algorithm: "md5",
-          });
-          url.searchParams.set("props", hashedProps);
-          span.setAttributes({
-            hash_size_bytes: hashedProps.length * 2,
-          });
-        } catch (e) {
-          span.recordException(e);
-          throw e;
-        } finally {
-          timing?.end();
-          span.end();
+        const resolveChainString = FieldResolver.minify(resolveChain)
+          .toString();
+        const revisionID = await release?.revision() ?? undefined;
```

Comment:
import { Context } from "deco/deco.ts";
...
// within the func
const context = Context.active();
const release = context.release;
//

the release changes based on the context